generators/entity-client/templates/react/src/main/webapp/app/entities/_entity-delete-dialog.tsx

```diff
@@ -0,0 +1,99 @@
+<%#
+ Copyright 2013-2017 the original author or authors from the JHipster project.
+
+ This file is part of the JHipster project, see http://www.jhipster.tech/
+ for more information.
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+-%>
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { Modal, ModalHeader, ModalBody, ModalFooter, Button } from 'reactstrap';
+import { Translate, ICrudGetAction, ICrudDeleteAction } from 'react-jhipster';
+import { FaBan, FaTrash } from 'react-icons/lib/fa';
+
+import { getEntity, deleteEntity } from './<%= entityFileName %>.reducer';
+
+export interface I<%= entityReactName %>DeleteDialogProps {
```

Comment:
The names are really long

Reply:
ya not sure what to do as we cant strip the props, and how much ever we try there will be some case where name is too long, let me see what can be done